packages/foundry/contracts/test/utils/TestConstants.sol

```diff
@@ -45,6 +45,9 @@ contract TestConstants {
         addresses[MAINNET][TIMELOCK] = 0x3b870db67a45611CF4723d44487EAF398fAc51E3;
         addresses[MAINNET][CAULDRON] = 0xc88191F8cb8e6D4a668B047c1C8503432c3Ca867;
         addresses[MAINNET][LADLE] = 0x6cB18fF2A33e981D1e38A663Ca056c0a5265066A;
+        addresses[TENDERLY][TIMELOCK] = 0x3b870db67a45611CF4723d44487EAF398fAc51E3;
+        addresses[TENDERLY][CAULDRON] = 0xc88191F8cb8e6D4a668B047c1C8503432c3Ca867;
+        addresses[TENDERLY][LADLE] = 0x6cB18fF2A33e981D1e38A663Ca056c0a5265066A;
```

Comment:
I've solved this [elsewhere](https://github.com/yieldprotocol/strategy-v2/blob/6bdaee346e486fb2b339a7c0397435130c197821/test/harness/OrchestrationHarness.t.sol#L33) by having an `RPC` variable with the RPC (which could be tenderly) and a `NETWORK` variable that is used for the addresses, but not to fork. Mine is not the cleanest approach, but it's a bit better.